docker/DOCKERHUB.md

```diff
@@ -0,0 +1,118 @@
+_Note: this is the source of truth for DHIS2 Docker images. Any changes to this document need to be
+manually synced to Dockerhub since we do not have a paid account._
+
+# Quick reference
+
+-	**Maintained by**:  
+	[the DHIS2 core team](https://github.com/dhis2/dhis2-core)
+
+-	**Where to get help**:  
+	[the DHIS2 Community of Practice - Tag docker](https://community.dhis2.org/tag/docker)
+
+# Supported tags
+
+* [`dhis2/core`](https://hub.docker.com/r/dhis2/core) - images of the release and release-candidate
+  DHIS2 versions. These images represent the **stable** DHIS2 versions, meaning they won't be
+  rebuilt in the future.
+
+* [`dhis2/core-dev`](https://hub.docker.com/r/dhis2/core-dev) - images of _the latest development_
+  DHIS2 versions - branches `master` (tagged as `latest`) and the previous 3 supported major
+  versions. Image tags in this repository will be overwritten multiple times a day.
+
+* [`dhis2/core-canary`](https://hub.docker.com/r/dhis2/core-canary) - images of _the latest daily
+  development_ DHIS2 versions. We tag the last `core-dev` images for the day and add an extra tag
+  with a "yyyyMMdd"-formatted date, like `core-canary:latest-20230124`.
+
+* [`dhis2/core-pr`](https://hub.docker.com/r/dhis2/core-pr) - images of PRs made from
+  https://github.com/dhis2/dhis2-core and not from forks. As forks do not have access to our
+  organizations/repos secrets.
+
+## Multi-architecture images
+
+Multi-platform images for `linux/amd64` and `linux/arm64` are published starting with
+
+* 2.39.1-rc
+* the release after 2.38.2.0
+* the release after 2.37.9
+
+# What is DHIS2?
+
+[DHIS2](https://dhis2.org/about) is an open source, web-based platform most commonly used as a
+health management information system (HMIS). It allows for data capture through clients ranging from
+web browsers, Android devices, Java feature phones and SMS. DHIS2 features data visualization apps
+for dashboards, pivot tables, charting and GIS. It provides metadata management and configuration.
+The data model and services are exposed through a RESTful Web API.
+
+# How to use this image
+
+## Using Docker Compose
+
+The easiest way to get familiar with DHIS2 in Docker is to start running it locally using [Docker
+Compose](https://docs.docker.com/compose/install/).
+
+https://github.com/dhis2/dhis2-core/blob/master/docker-compose.yml has an up-to-date
+docker-compose.yml that you can run using with
+
+```sh
+DHIS2_IMAGE=dhis2/core-dev:latest docker compose up
+```
+
+The above will start the latest development version of DHIS2 with PostgreSQL.
+
+[dhis-core/README](https://github.com/dhis2/dhis2-core) has more details on how to run DHIS2 using
```

Comment:
The link here can be `https://github.com/dhis2/dhis2-core#readme` to bring the focus directly to the README.